src/vs/workbench/contrib/mcp/common/mcpService.ts

```diff
@@ -153,13 +192,95 @@ export class McpService extends Disposable implements IMcpService {
 		}));
 	}
 
+	public async activateExtensionServers(): Promise<void> {
```

Comment:
Having this here feels wrong ;-) Extension activation etc should be contained in the host or collection but not leak into the "pure" service